common/app/common/jobs.scala

```diff
@@ -5,10 +5,12 @@ import java.util.TimeZone
 import org.quartz.impl.StdSchedulerFactory
 import org.quartz._
 import play.api.Mode.Test
+
 import scala.collection.mutable
 import play.api.{Environment, Play}
 
 import scala.concurrent.Future
+import scala.concurrent.duration.Duration
```

Comment:
good suggestion I didn't know about that way... if "seconds" was checked at compile time (i.e. a constant or something) i would, however as a magic string I dislike this even more! 😞